src/report-portal.js

```diff
@@ -396,6 +396,7 @@ class ReportPortal {
                 launchUuid: this.launch.id,
                 status: status,
                 endTime: time,
+                description: process.env.S3_BUCKET_URL ? `[Test Recordings](${encodeURI(process.env.S3_BUCKET_URL)})` : ''
```

Comment:
is the description field the general launch description (which is in use) or  test/fixture level?